ci/azure-win-installer-upload.yml

```diff
@@ -17,6 +17,8 @@ steps:
   - task: BatchScript@1
     inputs:
       filename: ci/win_test_installer.cmd
+    env:  
+      SKIP_ASSOCIATION_TEST: true # This test sometimes fails, moreover on Azure. It's run in GHA WindowsInstaller CI
```

Comment:
**If this is running on Azure**, it's OK to skip. But otherwise I would say this is important test because it checks if installed version can be launched without error (IIRC).

Reply:
Well that's the point, on Azure it fails more. But we will still test on GitHub Actions (master, release, PRs). So testing still done, but only once.